### PR TITLE
- Adds post-write hook to apply xslt; adds xslt; adds sample mappings…

### PR DESCRIPTION
**Pull request by jpeak5**
@mjordan We have addressed the issues you raised in PR 214.
This PR and PR 218 subsume PR  214.

- Adds xslt post-write hook
- Adds sample config/mapping files using it
- Adds a README that you may find useful in the mik wiki, describing the installation of saxon and how to use it with xslt post-write hook.

Thanks!

### CODE REVIEW

**Comment by mjordan** [comment]
@jpeak5 sweet, thanks! Let us smoke test this by end of the week and unless we see any problems we'll merge. This is an awesome contribution, thanks again.

File: extras/scripts/postwritehooks/apply_xslt.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * Works for single-file CONTENTdm and CSV import packages as well as
+ * newspaper issue packages, and can be extended to handle the MODS.xml
+ * files created by other MIK toolchains.
+ */
+
+require 'vendor/autoload.php';
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use GuzzleHttp\Client;
```
Comment (mjordan) [inline extras/scripts/postwritehooks/apply_xslt.php]: Sorry I didn't notice this earlier, but as far as I can tell you don't use Guzzle in this script. Can you remove this line?

File: extras/scripts/postwritehooks/apply_xslt.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Post-write hook script for MIK that validates MODS XML files.
+ * Works for single-file CONTENTdm and CSV import packages as well as
+ * newspaper issue packages, and can be extended to handle the MODS.xml
+ * files created by other MIK toolchains.
+ */
```
Comment (mjordan) [inline extras/scripts/postwritehooks/apply_xslt.php]: Can you update this comment so it better describes what this script does?

File: extras/scripts/xslt-postwrite.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+This post-write hook requires the Saxon HE processor. Follow the installation instructions, and place the file saxon9he.jar in your mik top-level folder. http://www.saxonica.com/html/documentation/about/gettingstarted/gettingstartedjava.html
+
```
Comment (mjordan) [inline extras/scripts/xslt-postwrite.md]: I had to think about this for a second. Can I suggest changing it to "place the file saxon9he.jar in your mik top-level MIK folder"?

**Comment by mjordan** [comment]
@jpeak5 smoke test worked great! I made three very small comments that are just cleanup. If you don't mind making those I'll merge your PR into master.

**Comment by flummingbird** [comment]
@mjordan Should be ready to go.

**Comment by mjordan** [comment]
Sweet... @MarcusBarnes any objections to merging this PR into master? If not, please do so.

**Comment by MarcusBarnes** [comment]
No objections.  Thank you @jpeak5 and @mjordan.  Merging now....

**Pull request merged**